src/commands/stepfunctions/helpers.ts

```diff
@@ -146,7 +152,7 @@ export type StatesType = Record<string, StepType>
 export type StepType = {
   Type: string
   Parameters?: ParametersType
-  Resource: string
+  Resource?: string
```

Comment:
Need to update this since I just learned that `Choice` doesn't have `Resource` field. Good that we catch this first.
```
    "ChoiceStep": {
      "Type": "Choice",
      "Comment": "some choice to make",
      "Choices": [
        {
          "Variable": "$.Comment",
          "StringEquals": "Insert your JSON here",
          "Next": "InvokeLambda"
        }
      ]
    },
```